lein-eftest/src/leiningen/eftest.clj

```diff
@@ -8,7 +8,15 @@
   {:dependencies '[[eftest "0.3.1"]]})
 
 (defn- report-namespace [project]
-  (-> project :eftest (:report 'eftest.report.progress/report) namespace symbol))
+  (if-let [reporter (get-in project [:eftest :report])]
+    (try
+      (-> reporter namespace symbol)
+      (catch NullPointerException cause
+        (throw (ex-info (str "eftest :report must be a fully-qualified reporter "
+                             "function!")
+                        {:reporter reporter}
+                        cause))))
+    'eftest.report.progress/report))
```

Comment:
I was caught out by an NPE here so thought I'd catch that and throw a more informative exception. It'd maybe be better to use something like clojure.spec to validate the options in someone's `project.clj` but I don't think we can assume Clojure 1.9 will be available everywhere eftest is used.